lib/features/pfis/add_pfi_confirmation_page.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:didpay/features/pfis/pfi.dart';
+import 'package:didpay/features/pfis/pfis_notifier.dart';
+import 'package:didpay/l10n/app_localizations.dart';
+import 'package:didpay/shared/pending_page.dart';
+import 'package:didpay/shared/success_page.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+
+class AddPfiConfirmationPage extends HookConsumerWidget {
+  final String did;
+
+  const AddPfiConfirmationPage({
+    required this.did,
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final addPfiState = useState<AsyncValue<Pfi>>(const AsyncLoading());
+
+    useEffect(
+      () {
+        _addPfi(context, ref, addPfiState);
+        return null;
+      },
+      [],
+    );
+
+    return Scaffold(
+      body: SafeArea(
+        child: addPfiState.value.when(
+          data: (pfi) => SuccessPage(text: Loc.of(context).pfiAdded),
+          loading: () => PendingPage(text: Loc.of(context).addingPfi),
+          error: (error, _) => Text('Error: $error'),
+        ),
+      ),
+    );
+  }
```

Comment:
<img width="484" alt="image" src="https://github.com/TBD54566975/didpay/assets/4887440/e256ea7d-269e-4f3b-bdd1-bf358a4be335">

also not important in this PR but wonder if this is necessary? or if we can just redirect back to where the user came from?

Reply:
so after the pfi has been successfully added, it goes back to `AppTabs` right away?